Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -46,19 +46,7 @@ public OrderTicket Buy(Symbol symbol, int quantity)
         /// Buy Stock (Alias of Order)
         /// </summary>
         /// <param name="symbol">string Symbol of the asset to trade</param>
-        /// <param name="quantity">double Quantity of the asset to trade</param>
-        /// <seealso cref="Buy(Symbol, int)"/>
-        public OrderTicket Buy(Symbol symbol, double quantity)
```

Comment:
Continuing from above, I could, add methods something like "OrderByDecimal(symbol, decimal)" and the overload problem goes away. However, I dislike the incongruity this introduces to the api.